packages/compiler-cli/src/ngtsc/typecheck/src/type_check_block.ts

```diff
@@ -1125,6 +1125,7 @@ class TcbExpressionTranslator {
       return result;
     } else if (ast instanceof MethodCall && ast.receiver instanceof ImplicitReceiver) {
       // Resolve the special `$any(expr)` syntax to insert a cast of the argument to type `any`.
+      // `$any(expr) -> expr as any`
```

Comment:
If you want to keep this comment I would introduce two more backticks:

```suggestion
      // `$any(expr)` -> `expr as any`
```